xbmc/cores/dvdplayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -45,6 +45,8 @@ CDVDInputStream* CDVDFactoryInputStream::CreateInputStream(IDVDPlayer* pPlayer,
 {
   CFileItem item(file.c_str(), false);
 
+  item.SetMimeType(content);
```

Comment:
this is a sensitive region. i only have a tablet and can't have a closer look. please wait with merge